index.js

```diff
@@ -1,15 +1,45 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = []
+    this.lenght = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    let index = 0 ;
+    while (index < this.lenght && item > this.item[index]){
+      index ++
+    }
+    this.items.splice(index, 0, item);
+    this.lenght++;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos < 0 || pos >= this.length) {
+      throw new Error("OutOfBounds");
+    }
+    return this.items[pos];
+  }
 
-  max() {}
+  max() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    }
+    return this.items[this.length - 1];
```

Comment:
Tambien aqui lo que podemos hacer es utilizar la función max() retorna el valor máximo presente en la lista. Para hacer esto 🔥 , utiliza el método Math.max(...this.items), que toma una cantidad variable de argumentos y devuelve el valor más grande entre ellos. En este caso, el operador de propagación ... se utiliza para pasar los elementos individuales de la lista como argumentos a la función Math.max().

```
return Math.max(...this.items)
```